packages/eslint-config-default/package.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "@wordpress/eslint-config-default",
+	"version": "1.0.0",
+	"description": "Default Eslint config for WordPress development",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"eslint"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/eslint-config-default/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"dependencies": {
+		"babel-eslint": "^8.0.3",
+		"eslint-config-wordpress": "^2.0.0",
+		"eslint-plugin-jsx-a11y": "^6.0.2",
+		"eslint-plugin-react": "^7.7.0",
+		"eslint-plugin-wordpress": "git://github.com/WordPress-Coding-Standards/eslint-plugin-wordpress.git#1774343f6226052a46b081e01db3fca8793cc9f1"
```

Comment:
@ntwb - can we publish this plugin to npm to avoid using git commit?

Reply:
Can we move all these *-wordpress repos here?

Reply:
I'll get this published once some of the coding standards ambiguities have been resolved

Reply:
I copied all rules and other settings from this repository.